examples/js/modules/esmodule.nim

```diff
@@ -0,0 +1,15 @@
+# import * from 'xyz'
+proc esImportAll*(nameOrPath: cstring) {.importcpp: "import * from #".}
+
+# import xyz from 'xyz'
+proc esImportDefault*(name: cstring, nameOrPath: cstring) {.
+    importcpp: "import # from #".}
+
+# import { default as abc } from 'xyz'
+proc esImportDefaultAs*(name: cstring, nameOrPath: cstring) {.
+    importcpp: "import { default as # } from #".}
+
+# import { x } from 'xyz'
+proc esImport*(name: cstring, nameOrPath: cstring) {.
+    importcpp: "import { # } from #".}
+
```

Comment:
These proc are useful, maybe actually add them to the JS std lib modules on a separate PR ?.

Reply:
I'm adding ES import/export interop helpers in a separate module for now. Just showcasing some ideas here. This PR will be in a state of flux for a while. Thanks for taking the time to review it.
Please make more suggestions for improvement.